mixer/pkg/server/server.go

```diff
@@ -30,6 +30,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
+	accessLogGRPC "github.com/envoyproxy/go-control-plane/envoy/service/accesslog/v2"
```

Comment:
nit: here and elsewhere we can drop `GRPC` from the import short name. you can use `pb` suffix to match others, or just drop.